toolkit/tools/imagecustomizerapi/mountidentifier.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerapi
+
+import "fmt"
+
+// MountIdentifierType indicates how a partition should be identified in the fstab file
+type MountIdentifierType string
```

Comment:
Are any of these types already defined in the Mariner toolkit since they have some overlap with these?

Reply:
Yeah, a lot of this has direct equivalents in the new-image config. But the image customizer guarantees back-compat. Whereas imager does not. So, the types are kept separate.